binaryset/binary_set.h

```diff
@@ -0,0 +1,22 @@
+#include <stdio.h>
+#include <stdlib.h>
+ 
+typedef int (*Comp)(const void*, const void*);
+
+struct BinarySetElement
```

Comment:
Private structs should follow the public structs' conventions and additionally they should have `_` (underscore) at the end of the name.
```c
typedef struct {
   // ...
} rtw_bset_elem_;
```